spec/index.bs

```diff
@@ -2400,10 +2400,10 @@ interface GPUCommandEncoder {
 
     void resolveQuerySet(
         GPUQuerySet querySet,
-        GPUSize32 queryFirstIndex,
+        GPUSize32 firstQuery,
```

Comment:
this isn't consistent with `queryIndex` we pass to `beginOcclusionQuery`

Reply:
We talked about `firstQueryIndex` but it could be misinterpreted as "first (query index)" instead of "(first query) index". We could use "start" instead of "first" but it would diverge from other naming.

Personally I think firstQuery is fine, anyway.

Reply:
I believe the most consistent name would be `startIndex`

Reply:
If we want to know the meaning of the parameter from the name, `startIndex` can be better represent what we want to pass. Otherwise, I think `firstQuery` or `firstQueryIndex` also make sense, we will explain in the spec that it specifies an index of first query to resolve.

Reply:
Note that in `beginOcclusionQuery` we don't have the `GPUQuerySet` in the same call, so we need the extra context. In `resolveQuerySet` the `GPUQuerySet` is right before `firstQuery` so I think context makes it clear what it is.

Reply:
there is only so much we can bikeshed :)
given no opposition from the editors, let's just land this